### ISSUE TITLE
No new cards being added

### ISSUE THREAD

**Issue by bobtheuberfish** [bobtheuberfish/chiriboga#154]
I have joined another in-progress Netrunner project with a larger team. This new project also has a play-against-AI feature but it is in early development at time of writing this. It is not web-based (code is C#) but theoretically it can be built for various platforms including browser.

I will still maintain Chiriboga in terms of handling issues and fixing bugs but I don't intend to add new cards. If you are interested in being involved in some way, please PM me on Discord, you can find me via Green Level Clearance.

**Comment by NethawkNY** [comment]
Any news on this new project?

**Comment by bobtheuberfish** [comment]
The new project is fully functioning but needs testing and polishing. If you would like to help with testing, you can catch us in the Green Level Clearance server on Discord, in the #tools-development channel.

**Comment by NethawkNY** [comment]
Thanks so much. I’ve had the game for years and finally started playing on chiriboga. I don’t do any programming but would be willing to help any way I can.

**Comment by bobtheuberfish** [comment]
Programming skills not needed, it would just help to have someone else playing the game against AI and reporting bugs, crashes, and bad AI decisions.

**Comment by NethawkNY** [comment]
Is there a section on discord on how to start? On Feb 5, 2025, at 3:52 AM, bobtheuberfish ***@***.***> wrote:﻿
Programming skills not needed, it would just help to have someone else playing the game against AI and reporting bugs, crashes, and bad AI decisions.

**Comment by Omnibit** [comment]
I can play & report if that's all it takes ^__^

**Comment by bobtheuberfish** [comment]
Certainly! You can find our team via Discord, we lurk in the tools-development channel on the Green Level Clearance server.

**Comment by JeffSkyrunner** [comment]
Hi!
How is this project called, and where we can find it? I am pretty new on discord...

**Comment by sboosali** [comment]
@JeffSkyrunner desktop Chiriboga seems to be called Vigeous or MOPUS: http://vigeous.com/

**Comment by JeffSkyrunner** [comment]
@sboosali thank you! Is there a Github project or it's just download... I can't find any source code here, but maybe it's under a different name?

**Comment by bobtheuberfish** [comment]
The source code is currently not public, ask around on GLC if you would like to be invited. A progress update on the new project - I've done the AI code for all of System Gateway and half of Elevation.

**Comment by lukastom** [comment]
I just want to say that Chiriboga is great because it can easily be played on iPad. But the new project will work only on PCs. It would be good to have both, something like a mobile version and also the desktop app. Anyways, thank you for this great work. Really helps me as a noob to learn the game!

**Comment by nalanbar** [comment]
@bobtheuberfish any chance of a Mac port? I see linux and windows. If you need a Mac user to help, I am glad to.

**Comment by bobtheuberfish** [comment]
@nalanbar We would love someone to help port to Mac (and web, if you're up for it!) Please message Vigeous or bobthefish, you can find us in the tools-development channel of Green Level Clearance.
